scripts/approve-fingerprint-sudo.sh

```diff
@@ -5,10 +5,13 @@
 # To be able to use the mac touchbar for authorizing the 'sudo' command in terminal
 # This will persist through software updates unlike changes directly made to '/etc/pam.d/sudo'
 # Copied from: https://apple.stackexchange.com/a/466029
-# TODO: Need to ensure that TouchId hardware is present before running this script
 
 type is_file &> /dev/null 2>&1 || source "${HOME}/.shellrc"
 
+if ! ioreg -c AppleBiometricSensor | grep -q "AppleBiometricSensor"; then
+  warn "Touch ID hardware is not detected. Skipping configuration."
```

Comment:
After warning, don't you also want to return from the script? Or else, the actual code starting from line 15 will need to be put inside an `else` block. I would prefer the 1st option of using a `return`

Reply:
What about exit 1? it will exit if no touchID hardware detected

Reply:
My last comment is missing - pushed new commit with RETURN since this script is sourced in other scripts